Server/src/services/helpers/validationSchema.ts

```diff
@@ -59,9 +59,9 @@ const referenceMaterialSchema =
 
 export const validationSchema = Yup.object().shape({
     reference: referenceValidationSchema,
-    ["dataset name"]: Yup.string().strict().required(requiredMessage('Dataset Name')),
+    dataset_name: Yup.string().strict().required(requiredMessage('Dataset Name')),
     material: Yup.array().of(referenceMaterialSchema),
-    ["data type"]: Yup.string().strict().required(requiredMessage('Dataset Type')),
+    data_type: Yup.string().strict().required(requiredMessage('Dataset Type')),
```

Comment:
This hasn't been changed in our testData file in the Server's tests folder. Please change that

Reply:
Done